xrm.jsonc

```diff
@@ -0,0 +1,8 @@
+{
+  "aadClient": "106f1a90-7645-46b8-b5fa-cf85de7d480b",
```

Comment:
Do we want to leave this in here, or is it ok since it's not super useful without the credentials?

Reply:
They are safe to have here, but having the file here in the first place was a side effect of testing, so I'll remove it